events/conn.go

```diff
@@ -20,13 +20,19 @@
 package events
 
 import (
+	"context"
 	"fmt"
-	"github.com/nats-io/nats.go"
+	"sync/atomic"
 	"time"
+
+	"github.com/nuts-foundation/nuts-node/vcr/log"
+
+	"github.com/nats-io/nats.go"
 )
 
 // Conn defines the methods required in the NATS connection structure
 type Conn interface {
+	Close()
```

Comment:
Close not called by any one?
Which adds the question, who is going to call close? The events engine?

Reply:
> who is going to call close? The events engine?

Yes, I think that makes sense. Will fix